examples/material_gallery/lib/demo/widget_demo.dart

```diff
@@ -4,12 +4,22 @@
 
 import 'package:flutter/material.dart';
 
+typedef Widget TabBarSelectionBuilder(BuildContext context, Widget child);
+
 class WidgetDemo {
-  WidgetDemo({ this.title, this.routeName, this.tabBarBuilder, this.floatingActionButtonBuilder, this.builder });
+  WidgetDemo({
+    this.title,
+    this.routeName,
+    this.tabBarBuilder,
+    this.tabBarSelectionBuilder,
+    this.floatingActionButtonBuilder,
+    this.builder
+  });
 
   final String title;
   final String routeName;
   final WidgetBuilder tabBarBuilder;
+  final TabBarSelectionBuilder tabBarSelectionBuilder;
```

Comment:
Sure, I'll change that. At some point I think it would be worth revisiting the WidgetDemo and GalleryPage types. They don't work well for demoing tabs, toolbars, etc.